index.ios.js

```diff
@@ -40,7 +41,12 @@ var MusicControl = {
     NativeMusicControl.enableBackgroundMode(enable)
   },
   setNowPlaying: function(info){
-    NativeMusicControl.setNowPlaying(info)
+    // Check if we have an android asset from react style image require
```

Comment:
Oops, copy and paste strikes again, sorry.  This allows the loading of artwork from RN's require(), which was previously not supported on iOS.